builder/googlecompute/config.go

```diff
@@ -116,7 +116,8 @@ type Config struct {
 	IAPConfig `mapstructure:",squash"`
 	// Skip creating the image. Useful for setting to `true` during a build test stage. Defaults to `false`.
 	SkipCreateImage bool `mapstructure:"skip_create_image" required:"false"`
-	// The architecture of the resulting image. Defaults to "x86_64".
+	// The architecture of the resulting image. Defaults to the empty string,
+	// the original image's architecture will carry over by default.
```

Comment:
```suggestion
	// GCP will use original image's architecture will carry over by default.
```

I feel like this makes it more clear that we're not sending the default architecture, Google just uses it when unset